src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -14,103 +14,150 @@ resources:
-* [Demo Deployer](https://github.com/newrelic/demo-deployer) - This is the engine that drives all the automation. For convenience, you'll clone this repo and build a docker image. In future iterations, we'll publish the image in docker hub or someplace similar. 
-* [Demo Nodetron](https://github.com/newrelic/demo-nodetron) - This is the simple app that will be provisioned on a host. It will give you a simple web page and an API that you can hit with a tool like [Postman](https://www.postman.com/downloads/) depending on how much traffic you want to generate. 
-* [Demo New Relic Instrumentation](https://github.com/newrelic/demo-newrelic-instrumentation) - This is the project that we use to layer in the instrumentation for the infrastructure and the application. 
+Throughout this guide, you'll use three open source projects to automatically set up, instrument, and teardown your infrastructure:
+
+- [Demo Deployer](https://github.com/newrelic/demo-deployer): The engine that drives all the automation
+- [Demo Nodetron](https://github.com/newrelic/demo-nodetron): A Node.js app that you will provision on your host. It will present a web page and an API, which you will use to generate traffic.
+- [Demo New Relic Instrumentation](https://github.com/newrelic/demo-newrelic-instrumentation): A project that you will use to instrument your host and application
+
+Each project is distinct, and encapsulates its own behavior. This maintains a separation of concerns and allows you to compose complex scenarios using a modular approach.
+
+<Tip title="Collaborate with us">
+
+The Demo Deployer is currently in development. Because of our commitment to open source, we are working in the open early in the process and invite you to collaborate with us. Drop any thoughts and comments in the [Build on New Relic](https://discuss.newrelic.com/c/build-on-new-relic) support thread, and let us know what you think!
+
+</Tip>
 
-We kept each project distinct to encapsulate its behavior. The idea being that we can simply add more behaviors and then weave them in to assemble more complex scenarios. Cool, right?
+Now that you have a high-level understanding of the tools you will use, you can begin by building the Demo Deployer!
 
 <Steps>
 
 <Step>
 
-### Build the Deployer
-Let's get the deployer built. Essentially, this is cloning the repo and building the docker image. If you don't have docker installed, you can download it [here](https://docs.docker.com/get-docker/). 
+## Build the Deployer
 
-* Clone the deployer repo
+To use the Demo Deployer, you will first need to clone its repository from GitHub. After you have a local copy of the code, you will use it to build a Docker image.
 
-```bash
-git clone https://github.com/newrelic/demo-deployer.git $HOME/deployer
+<Important>
+
+This guide requires that you have [Docker](https://docs.docker.com/get-docker/) installed on your local machine.
```

Comment:
Having Docker installed locally should be added to the "Before you begin" section as a prerequisite. Then you won't need this callout. (that way, readers get the info ahead of time.)